app/views/groups/index.html.erb

```diff
@@ -10,22 +10,14 @@
   <div class="h3">
     <%= render GroupNameComponent.new(group: group) %>
     <% if @user.admin_in?(group)%>
-      <%= link_to " ", edit_group_path(group), class: "badge bg-primary bi bi-pencil-fill" %>
+      <%= link_to " ", edit_group_path(group), class: "badge bg-primary bi bi-pencil-fill", style: "z-index:2; position:relative" %>
```

Comment:
Z-indexes 0 to 3 are used by bootstrap classes for different states of elements. (See [their approach to z-index scales](https://getbootstrap.com/docs/5.2/extend/approach/#z-index-scales))

Maybe we should use some higher index, like 4 or 10 (the latter one being more satisfying imo).

Reply:
Is there anything failing right now?
Would 10 render on top of e.g. modals?

Reply:
Right now, nothing is failing. But – that's why I linked the bootstrap documentation – if we ever decide to change the style of something under the button (e.g., on focus) this styling will be shown above the button (z-index 3).

You can also read in the documentation that all other bootstrap classes (including modals) have z-indexes of 1000 and higher.

Maybe we could create a custom CSS class for all buttons that should be on top of stretch links.

Reply:
[f2447dd](https://github.com/hpi-swt2/bookkeeper-portal-red/pull/360/commits/f2447dd6e257782cc3561ed67c283233041645ed) should resolve this and all other change requests